module2/HangMan.rb

```diff
@@ -0,0 +1,47 @@
+
+def hangman(word)
+    lifes = 5;
+    incomplete_word = Array.new(word.length, "_");  #new aaray of word size
+    incomplete_word[0] = word[0];                   #setting 1st word
+    incomplete_word[word.length-1] = word[word.length-1];   #setting lasr word
```

Comment:
You are only setting first and last letters. but in this game, words may have in between letters visible. consider that scenario also.